packages/components/src/DataTable/CustomPagination.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useMemo, useState } from "react";
```

Comment:
this is just for the use of DataTable.mdx from what I can see. It would be helpful to rename this and/or limit its exportability to reduce confusion

Reply:
Yeah, I tried to move this example to the playground. But it kept throwing some weird errors

Reply:
✅